code/game/gamemodes/miniantags/pulsedemon/pulsedemon.dm

```diff
@@ -119,8 +119,10 @@
 	/// A cyborg that has already been hijacked can be re-entered instantly.
 	var/list/hijacked_robots
 
-	/// Images currently being shown on the client.
-	var/list/images_shown = list()
+	/// Images of cables currently being shown on the client.
+	var/list/cable_images = list()
+	/// Images of APCs currently being shown on the client.
+	var/list/apc_images = list()
```

Comment:
These need to to be cleared in `Destroy`
```dm
QDEL_LIST_CONTENTS(cable_images)
QDEL_LIST_CONTENTS(apc_images)
```
And in fact, every hard reference on the pulse demon needs to be nulled in `Destroy` (and/or deleted if it's needed). Vars such as `current_bot`, `current_cable`, or `current_weapon` for example.